### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by thekp**
**Overall change:** _Merging dependabot PRs._

**Code changes:**

This PR covers the following dependabot PRs: 
https://github.com/bbc/simorgh/pull/4218, https://github.com/bbc/simorgh/pull/4217, https://github.com/bbc/simorgh/pull/4216, https://github.com/bbc/simorgh/pull/4215, https://github.com/bbc/simorgh/pull/4214, https://github.com/bbc/simorgh/pull/4213, https://github.com/bbc/simorgh/pull/4212, https://github.com/bbc/simorgh/pull/4209, https://github.com/bbc/simorgh/pull/4211, https://github.com/bbc/simorgh/pull/4210, https://github.com/bbc/simorgh/pull/4208, https://github.com/bbc/simorgh/pull/4207

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "@bbc/psammead-amp-geo": "^1.1.2",
     "@bbc/psammead-assets": "^2.5.0",
     "@bbc/psammead-brand": "^5.0.5",
-    "@bbc/psammead-calendars": "^1.0.0",
+    "@bbc/psammead-calendars": "^2.0.0",
```
Comment (chris-hinds) [inline package.json]: This is a major version change. Are we happy to bring this in and has it been tested in Simorgh yet?

**Review by chris-hinds — COMMENTED** [review]
Just one question.

File: package.json (hunk 1 above)
Comment (hotinglok) [reply package.json]: I think so as [this pr](https://github.com/bbc/simorgh/pull/4254) was merged yesterday:
https://github.com/bbc/simorgh/blob/e28f15b351630b90562ed85bf34d9478d02253e5/package.json#L62

Comment (thekp) [reply package.json]: good spot, i'll merge latest into this branch 🏭

Comment (jamesdonoh) [reply package.json]: Can confirm was tested https://github.com/bbc/simorgh/pull/4254#issuecomment-542736211

**Review by hotinglok — APPROVED** [review]

**Review by FK78 — COMMENTED** [review]
CI failing.

**Comment by thekp** [comment]
@FK78 the merge to latest should hopefully fix that
Lighthouse failed and it's been disabled. Hopefully, it passes now

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**